src/main/resources/db/migration/V1_create_movies.sql

```diff
@@ -0,0 +1,9 @@
+DROP TABLE IF EXISTS movie CASCADE;
+CREATE TABLE movie
+(
+    id         INTEGER GENERATED BY DEFAULT AS IDENTITY,
+    legacy_id  INTEGER,
+    public_id  UUID,
+    movie_name VARCHAR(255),
+    PRIMARY KEY (id)
+);
```

Comment:
Utilizar comando COMMENT para comentário da tabela e solunas (no caso de bds que possuem a feature). Exemplo: 

COMMENT ON TABLE SCHEMA.MOVIE IS ' [NOT_SECURITY_APPLY] (descrição em ingles do que se trata a tabela)...';
COMMENT ON COLUMN SCHEMA.MOVIE.IDT_MOVIE IS ' [NOT_SECURITY_APPLY] primary key identifier, for the movie.';